lib/js/jslib.js

```diff
@@ -1757,11 +1757,30 @@ function _getInputValue(id) {
   var element = document.getElementById(id);
   DEBUG.assert(element != null, "invalid input node (id " + id + ")");
   DEBUG.assert(element.value != undefined, "invalid input value in id " + id)
-  return element.value;
+  if ((element.type != "radio" && element.type != "checkbox") || element.checked) {
+    return element.value;
+  } else {
+    return "";
+  }
 }
 
 var getInputValue = LINKS.kify(_getInputValue);
 
+// getRadioGroupValue : [String] -> String
+function _getRadioGroupValue(ids) {
+  let ptr = ids;
+  while ( LINKEDLIST.isNil(ptr) == false ) {
+    var element = document.getElementById(ptr._head);
```

Comment:
Use `const` rather than `var` here. The former is block-scoped, whilst the latter is function-global scoped.

Reply:
Again this was based on 
https://github.com/links-lang/links/blob/faa49cd09cc20429122c9def648b25cc3cab991b/lib/js/jslib.js#L1757

so should that be changed too?

Reply:
Using `const` over `var` for L1757 instance should not make much of a difference as the variable is function-global scoped regardlessly. However, `const` (or `let`) might help the compiler apply some optimisations, although in this specific case I don't think there is much room for optimisations.

Reply:
OK